src/test/java/spoon/test/filters/FilterTest.java

```diff
@@ -332,6 +335,8 @@ public void testOverriddenMethodFromAbstractClass() throws Exception {
 		final CtClass<AbstractTostada> aClass = launcher.getFactory().Class().get(AbstractTostada.class);
 
 		assertEquals(0, Query.getElements(launcher.getFactory(), new OverriddenMethodFilter(aClass.getMethodsByName("prepare").get(0))).size());
+		assertEquals(0, Query.query().then(new OverriddenMethodFilter()).list(aClass.getMethodsByName("prepare").get(0)).size());
```

Comment:
> Query.query().then(...)

yes, I do not like it too. But it was minimum code at the beginning. If you have better idea, then tell me please. Otherwise I suggest to wait little bit until I (or somebody else) start to use it and come with real experience and needs... I am also looking for a better way ...